### PR TITLE
feat: new homepage

### PR DESCRIPTION
**Pull request by v-racine**

### CODE REVIEW

**Comment by v-racine** [comment]
#Description
- Adding a new service for public-facing pages when users aren't signed in
- Adding some style to the homepage

# Checklist
 - [ ] builds
 - [ ] tests run
 - [ ] it's observable when deployed
 - [ ] added docstrings

File: src/services/usersProductsService.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class UsersProductsService {
```
Comment (w-h-a) [inline src/services/usersProductsService.js:1]: why can't we use the productService we already have?

**Review by w-h-a — CHANGES_REQUESTED** [review]
use the same service

**Comment by v-racine** [comment]
I did it!

File: src/app.js [hunk 2]
```diff
@@ -41,8 +40,7 @@ const AppFactory = (args) => {
   const healthService = new HealthService({ usersRepo });
   const usersService = new UsersService({ usersRepo });
   const productsService = new ProductsService({ productsRepo });
-  const usersProductsService = new UsersProductsService({ productsRepo });
-
+  ``;
```
Comment (w-h-a) [inline src/app.js:43]: what is this?

**Review by w-h-a — APPROVED** [review]

**Pull request merged**